solr/core/src/java/org/apache/solr/handler/SolrConfigHandler.java

```diff
@@ -1032,11 +1031,6 @@ public Boolean call() throws Exception {
     protected SolrResponse createResponse(SolrClient client) {
       return null;
     }
-
-    @Override
-    public String getRequestType() {
```

Comment:
If you remove this here then you should supply ADMIN in the constructor.  This underscores a point I make in my review here that it should be explicit; not an optional parameter.